crates/jit/src/code_memory.rs

```diff
@@ -81,16 +112,17 @@ impl CodeMemory {
         self.push_current(0)
             .expect("failed to push current memory map");
 
-        for (m, t) in &mut self.mmaps[self.published..] {
+        for CodeMemoryEntry { mmap: m, table: t } in &mut self.mmaps[self.published..] {
+            // Remove write access to the pages due to the relocation fixups.
```

Comment:
Another extreme nit:

```suggestion
            // Publish the function table (thus performing relocation fixups) before removing write access.
```